### ISSUE TITLE
Wikidata/wikipedia IDs?

### ISSUE THREAD

**Issue by danbri** [CrossRef/open-funder-registry#5]
Has anyone mapped these entries to Wikidata yet?

**Comment by Daniel-Mietchen** [comment]
Yes, [about 13k of them are in](https://query.wikidata.org/#SELECT%20DISTINCT%20%3Fitem%20%3FitemLabel%20%3Ffund_id%20WHERE%20%7B%0A%20%20%3Fitem%20wdt%3AP3153%20%3Ffund_id%20.%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D):

```SPARQL
SELECT DISTINCT ?item ?itemLabel ?fund_id WHERE {
  ?item wdt:P3153 ?fund_id .
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
}
```
